test/dataLabel/dataLabelManagerTest.ts

```diff
@@ -61,10 +62,19 @@ module powerbi.extensibility.utils.chart.dataLabel.test {
             });
 
             it("Get Label info - all values Provided", () => {
-                let result: IDataLabelInfo = labelManager.getLabelInfo({ maximumMovingDistance: 12 });
+                let result: IDataLabelInfo = labelManager.getLabelInfo({ maximumMovingDistance: 16 });
```

Comment:
Why did you change the value in the existent test?

Reply:
Changed it back.